### ISSUE TITLE
Question upon MSE loss

### ISSUE THREAD

**Issue by cliangyu** [pengzhiliang/MAE-pytorch#80]
Why MAE adopts MSE as loss function, while not BCE or something else?

**Comment by pengzhiliang** [comment]
MSE loss is more suitable for regression problems.
L1/smoth-l1 also works, I guess.
